src/index.ts

```diff
@@ -40,8 +40,13 @@ async function setEnvVars(): Promise<void> {
       core.exportVariable(k, v)
     }
   }
-  if (getExperimental()) set('MISE_EXPERIMENTAL', '1')
-  set('MISE_LOG_LEVEL', core.getInput('log_level') || 'info')
+  if (core.getBooleanInput('experimental')) set('MISE_EXPERIMENTAL', '1')
+
+  const logLevel = core.isDebug()
+    ? 'debug'
+    : core.getInput('log_level') || 'info'
+
+  set('MISE_LOG_LEVEL', logLevel)
```

Comment:
if the user didn't specify debug mode or input log_level we should not set `MISE_LOG_LEVEL` at all

Reply:
The old (current) code defaults to info when debug level is not set so I kept that behaviour. I agree that maybe it makes more sense to not set the `MISE_LOG_LEVEL` when the user has not set `log_level`.

I'll make that change.